controllers/bookController.js

```diff
@@ -1,18 +1,56 @@
 const Book = require("../models/bookModel");
+const AppError = require("../utils/appError");
```

Comment:
Also, you are too forward. This PR contains too much. Limit this PR to just signup and login